packages/google_maps_flutter/google_maps_flutter_android/README.md

```diff
@@ -70,8 +70,14 @@ AndroidMapRenderer mapRenderer = AndroidMapRenderer.platformDefault;
   }
 ```
 
-Available values are `AndroidMapRenderer.latest`, `AndroidMapRenderer.legacy`, `AndroidMapRenderer.platformDefault`.
-Note that getting the requested renderer as a response is not guaranteed.
+NOTE: `AndroidMapRenderer.platformDefault` corresponds to `AndroidMapRenderer.latest`.
```

Comment:
Doesn't this depend on the device and server behavior? I wouldn't think we could say this at the plugin level.

Reply:
No, the client library (that we dep in via gradle) controls this logic and default is now interpreted as latest.